libkbfs/folder_branch_ops.go

```diff
@@ -3233,9 +3368,13 @@ func (fbo *folderBranchOps) undoMDUpdatesLocked(ctx context.Context,
 				fbo.getCurrMDRevisionLocked(lState)}
```

Comment:
Oops, I should change this check to use the error returned by `setHeadPredecessorLocked`, or something.